### PR TITLE
[DE] German translation

### PR DESCRIPTION
**Pull request by csagataj2**
Start for German translation

de-01
de-02
de-03
de-04

### CODE REVIEW

File: .github/qr-codes-de.tex [hunk 1]
```diff
@@ -0,0 +1,12 @@
+\bigbreak
+
+\begin{multicols}{2}
+\centering
+\includegraphics[width=0.8\linewidth]{\qr/github.png}\\
+Scan to open \textbf{GitHub} repository.
```
Comment (qwrtln) [inline .github/qr-codes-de.tex:6]: Please make it an actual German 😆

**Review by qwrtln — CHANGES_REQUESTED** [review]
1. Update readme
2. Make qr codes an actual German
3. Add the rest of po files - you can use github action for this
4. Update tools/clean.sh script

**Comment by qwrtln** [comment]
po4a check

**Comment by csagataj2** [comment]
po4a commit

**Comment by qwrtln** [comment]
Just the QR codes left.

**Comment by qwrtln** [comment]
Ah, sorry, the README entry for Germain is missing. Please add it.

**Comment by qwrtln** [comment]
Also, please send me the chosen German version of RULE BOOK REWRITE. I'll create a suitable image for you.

**Comment by markdermark** [comment]
> Also, please send me the chosen German version of RULE BOOK REWRITE. I'll create a suitable image for you.

RULE BOOK REWRITE ==> REGELWERK NEUGESTALTET

File: README.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Click in the table to download the most recent builds in the chosen language:
 | 🇫🇷 Français   |   ~95%   | [🇫🇷📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/main_fr.pdf) | [🇫🇷🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/printable_fr.pdf) |
 | 🇷🇺 Русский    |   ~39%   | [🇷🇺📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/main_ru.pdf) | [🇷🇺🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/printable_ru.pdf) |
 | 🇺🇦 Українська |   ~10%   | [🇺🇦📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/main_ua.pdf) | [🇺🇦🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/printable_ua.pdf)|
+| DE Deutsch    |   ~1%   | [DE📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/main_de.pdf) | [DE🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/printable_de.pdf)|
```
Comment (qwrtln) [inline README.md:17]:
```suggestion
| 🇩🇪 Deutsch    |   ~1%   | [🇩🇪📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/main_de.pdf) | [🇩🇪🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/printable_de.pdf)|
```

**Comment by qwrtln** [comment]
I added pictures. Look at the introduction. It looks like a weird mixture of German and French.

**Comment by markdermark** [comment]
> I added pictures. Look at the introduction. It looks like a weird mixture of German and French.

Yeah saw it too. So i directly added it.

File: translations/introduction.tex/de.po [hunk 3]
```diff
@@ -81,9 +81,9 @@ msgid ""
 msgstr ""
 "{2}\n"
 "\\textbf{Note} : In diesem Regelwerk werden Spiel- und Zubehörbegriffe in Großbuchstaben geschrieben.\n"
```
Comment (qwrtln) [inline translations/introduction.tex/de.po:83]:
```suggestion
"\\textbf{Note}: In diesem Regelwerk werden Spiel- und Zubehörbegriffe in Großbuchstaben geschrieben.\n"
```

File: translations/introduction.tex/de.po [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+#. type: Plain text
+#: sections/introduction.tex:7
+msgid ""
+"The continent of Antagarich is at war as several different Factions, led by "
+"their Heroes, battle for supremacy. Choose your Faction and Hero and banish "
+"your unruly enemies from these lands!"
+msgstr ""
+"Auf dem Kontinent Antagarich herscht Krieg. Unterschiedliche Fraktionen, "
+"angeführt durch Ihre Helden, kämpfen um die Vorherrschaft. Wähle auch du "
+"eine Fraktion und Held, um deine Gegner aus diesen Landen zu verbannen !"
```
Comment (qwrtln) [inline translations/introduction.tex/de.po:41]:
```suggestion
"eine Fraktion und Held, um deine Gegner aus diesen Landen zu verbannen!"
```

File: translations/introduction.tex/de.po [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+"  \\medskip\\\\\n"
+"  Conflicting rule changes on components follow this priority: Player Cards, Unit Cards, Town Boards, Mission Book, this rule book.\n"
+"}\n"
+"\\vfill"
+msgstr ""
+"{2}\n"
+"\\textbf{Anmerkung}: In diesem Regelwerk werden Spiel- und Zubehörbegriffe in Großbuchstaben geschrieben.\n"
+"\\textbf{Fettschrift} wird genutzt, um die Aufmerksamkeit auf wichtige Regel zu lenken.\n"
+"\\textit{Kursivschrift} wird für Beispiele verwendet.\n"
+"\\hyperlink{Heroes of Might and Magic III}{>Ein Brauner Internetlink} führt dich zu einem bestimmten Abschnitt im Regelwerk.\n"
```
Comment (qwrtln) [inline translations/introduction.tex/de.po:86]:
```suggestion
"\\hyperlink{Heroes of Might and Magic III}{Ein Brauner Internetlink} führt dich zu einem bestimmten Abschnitt im Regelwerk.\n"
```

Comment (qwrtln) [reply translations/introduction.tex/de.po:86]: Those are not internet links, those refer you just within the document.

**Review by qwrtln — APPROVED** [review]

**Pull request merged**